common/debug/valgrind/valgrind-libgda.supp

```diff
@@ -37,7 +37,7 @@
 #     if Free: name of free-ing fn)
 
 {
-	libgda permanant memory for gda_paramlist_dtd
+	libgda permanent memory for gda_paramlist_dtd
```

Comment:
I don't know enough of how valgrind works to evaluate whether the name of the suppression matters at all. So I don't know if it's safe to fix the name (though I would assume it is?).

Reply:
The first line is the name/description https://wiki.wxwidgets.org/Valgrind_Suppression_File_Howto#Suppression_file_syntax

It is effectively a comment.

Reply:
Ok, that's what I thought. I just wanted to be sure this string is not referenced anywhere else, like in command line parameters or in other suppression files (not necessarily part of the gnucash repo).